src/components/core/Input/Input.tsx

```diff
@@ -1,11 +1,13 @@
+/* eslint-disable react/display-name */
```

Comment:
This can be mitigated by setting `Input.displayName = 'Input'` if I remember correctly. Details [here](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/display-name.md)

However, pretty sure this is only for e.g. displaying in a debug render tree (for example, in the React dev tools extension), so not critical